ssl/ssl_sess.c

```diff
@@ -246,6 +246,12 @@ SSL_SESSION *ssl_session_dup(SSL_SESSION *src, int ticket)
     return NULL;
 }
 
+SSL_SESSION *SSL_SESSION_dup(SSL_SESSION *src) {
+    if (src == NULL)
```

Comment:
No.  We do not check for errors on NULL parameters.  That's a programming error.  The only NULL things we allow are for malloc and free wrappers, because the C library has specific semantics for those.